docker/build/Dockerfile

```diff
@@ -1,13 +1,15 @@
 # SPDX-FileCopyrightText: 2020 TQ Tezos <https://tqtezos.com/>
 #
 # SPDX-License-Identifier: LicenseRef-MIT-TQ
-FROM alpine:3.11
+FROM alpine:3.12
 RUN apk update
 RUN apk --no-cache --virtual add rsync git m4 build-base patch unzip \
   bubblewrap wget pkgconfig gmp-dev libev-dev hidapi-dev eudev-dev perl opam libusb-dev bash \
   autoconf automake libtool linux-headers cargo libffi-dev
 COPY ./build/build-libusb-and-hidapi.sh /build-libusb-and-hidapi.sh
 RUN /build-libusb-and-hidapi.sh
+# RUN wget https://sh.rustup.rs/rustup-init.sh && chmod +x rustup-init.sh && \
+#   ./rustup-init.sh --profile minimal --default-toolchain 1.44.0 -y
```

Comment:
Why is it commented out, what's the destiny of this code?

Reply:
hmm, I thought, I removed it :thinking: